src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml

```diff
@@ -253,3 +253,9 @@ sylius_fixtures:
                                 name: 'Winter sale'
                                 channels:
                                     - 'FASHION_WEB'
+                                        
+                catalog_promotion_rule:
+                    options:
+                        custom:
+                            for_variants:
+                                catalogPromotion: 'winter'
```

Comment:
you should also configure here rule configuration - array with `productVariants` codes